src/cpp/CMakeLists.txt

```diff
@@ -76,6 +76,7 @@ if(UNIX)
 
    # gcc hardending options (see: http://wiki.debian.org/Hardening)
    if(NOT APPLE)
+      add_definitions(-O2)
```

Comment:
Realize that this is extremely superstitious but I'd rather not change an optimization level that's been static for years right on the eve of shipping. Let's just live with the warnings then change this when development started back up again.